src/components/DropZone/tests/DropZone.test.tsx

```diff
@@ -473,6 +473,9 @@ function setBoundingClientRect(size: keyof typeof widths) {
         left: 0,
         bottom: 0,
         right: 0,
+        x: 0,
+        y: 0,
+        toJSON() {},
```

Comment:
Why is this needed?

Reply:
Probably to satisfy the type `Element['getBoundingClientRect']`:
https://github.com/Shopify/polaris-react/pull/2549/files#diff-ba67dc9a18e9f3acebe10b28d3397442R1029

Reply:
TS Updated their types and added these properties https://github.com/Shopify/polaris-react/pull/2237#discussion_r344419536